app/views/teachers/ConvertToTeacherAccountView.coffee

```diff
@@ -0,0 +1,138 @@
+RootView = require 'views/core/RootView'
+forms = require 'core/forms'
+TrialRequest = require 'models/TrialRequest'
+TrialRequests = require 'collections/TrialRequests'
+AuthModal = require 'views/core/AuthModal'
+storage = require 'core/storage'
+errors = require 'core/errors'
+User = require 'models/User'
+ConfirmModal = require 'views/editor/modal/ConfirmModal'
+
+FORM_KEY = 'request-quote-form'
+
+module.exports = class ConvertToTeacherAccountView extends RootView
+  id: 'convert-to-teacher-account-view'
+  template: require 'templates/teachers/convert-to-teacher-account-view'
+  logoutRedirectURL: null
+
+  events:
+    'change form': 'onChangeForm'
+    'submit form': 'onSubmitForm'
+    'click #logout-link': -> me.logout()
+
+  initialize: ->
+    if me.isAnonymous()
+      application.router.navigate('/teachers/signup', {trigger: true, replace: true})
+      return
+    @trialRequest = new TrialRequest()
+    @trialRequests = new TrialRequests()
+    @trialRequests.fetchOwn()
+    @supermodel.trackCollection(@trialRequests)
+
+  onLoaded: ->
+    if @trialRequests.size()
+      @trialRequest = @trialRequests.first()
+    if @trialRequest and @trialRequest.get('status') isnt 'submitted' and @trialRequest.get('status') isnt 'approved'
```

Comment:
This was copied from RequestQuoteView, which was added here:

Copied from RequestQuoteView since it seemed like it was effectively the same behavior. Should it be not included? Renamed? Removed from both?

https://github.com/codecombat/codecombat/commit/b495c581a7696c15a322711224aa6af009b0eacf